### DISCUSSION TITLE
Reason for `DiatonicNiLow`/`DiatonicNiHigh`

### DISCUSSION THREAD

**Discussion by basil** [neanes/neanes#307] [Q&A] [answered]
There are two diatonic Ni fthoras while all other diatonic fthoras have only one representation. Why is this? As far as I know, any diatonic fthora can be used in any octave.

**Comment by danielgarthur** [comment]
Are you asking about this software, or about the history of chant? I'm not entirely sure why the distinction was necessary. My guess would be that it was to forward the idea of the diatonic scale being diapason as opposed to diapente. Eight notes in the scale and so eight fthores. Or maybe there was confusion at some point in the past on how low Zo vs high Zo should be chanted (how flat they are, for example) and the distinction was necessary.

**Comment by basil** [comment]
I meant why are there two separate buttons in the UI for the same symbol. Is there a use case that is satisfied by having two buttons that isn't satisfied by having only one button?

**Comment by danielgarthur** [comment, marked as answer]
They are two separate symbols: 
<img width="73" alt="image" src="https://github.com/danielgarthur/neanes/assets/4132779/fec860fa-ded9-47b5-ac13-6f6f8599f112">

An example would be when sometimes the fthora of Ga is placed on Ni' to temporarily make the interval between Ni' Pa' Vou' 12-12, and then returning the Ni' to its normal place with a high Ni' fthora.

**Comment by basil** [comment]
Thank you, I never noticed that before.
